server/plugin/plugin.go

```diff
@@ -111,6 +113,29 @@ func NewPlugin() *Plugin {
 		"issue":         p.handleIssue,
 	}
 
+	var baseGithubEmojiMap = map[string]string{
+		"+1":         "+1",
+		"-1":         "-1",
+		"thumbsup":   "+1",
+		"thumbsdown": "-1",
+		"laughing":   "laugh",
+		"confused":   "confused",
+		"heart":      "heart",
+		"tada":       "hooray",
+		"rocket":     "rocket",
+		"eyes":       "eyes",
+	}
+
+	var githubEmojiMap = map[string]string{}
+	for systemEmoji := range model.SystemEmojis {
+		for mmBase, ghBase := range baseGithubEmojiMap {
+			if strings.HasPrefix(systemEmoji, mmBase) {
```

Comment:
What was the need for iterating on `model.SystemEmojis` map? Was it for supporting emojis with different skin tones?

Reply:
yes manoj